cookbooks/oceanic_extension/doc/oceanic_extension.tex

```diff
@@ -0,0 +1,60 @@
+Finally, because there are no inflow boundaries, we do not need to specify anything for the boundary conditions of the compositional fields.
+
+\lstinputlisting[language=prmfile]{cookbooks/oceanic_extension/doc/oceanic_extension_boundary_conditions.prm.out}
+
+
+\begin{figure}
+\centering
+\includegraphics[width=\textwidth]{cookbooks/oceanic_extension/doc/figure_initial_setup.png}
+\caption{\it Initial model setup. Top panel: compositional fields of oceanic crust (blue) and lithospheric mantle (green) and velocity field at time step 0. Bottom panel: initial random values of plastic strain and mesh. Isotherms are shown in white each 200 $\degree C$.}
+\label{fig:figure_initial_setup}
+\end{figure}
+
+\paragraph{Rheology}
+To model lithospheric deformation, we use a visco-plastic rheology, which takes into account both brittle and viscous deformation. To do that, we use the material model "visco plastic" and the viscous flow law "composite". In this way, the viscosity for each deformation mechanism (i.e, Drucker Prager yielding, diffusion creep, and dislocation creep) will be computed in each point of the grid and the lowest value will be used as effective viscosity. We define values of flow laws for diffusion and dislocation creep for each compositional field: gabbro flow law for the oceanic crust (Wilks and Carter, 1990) and dry olivine for the lithospheric mantle (Hirth and Kholstedt, 2003). Because we also use a compositional field for plastic strain, we need to define values also for this field. We set these parameters to values that give an unrealistic high viscosity, so that they will not be used to compute the effective viscosity. 
+In addition to the deformation mechanisms described above, we also allow for strain weakening. In this model, plastic weakening depends only on plastic strain. This is where we use the plastic strain compositional field that we defined before. By setting weakening factors to 0.5, friction and cohesion will be lowered by factor 2 between plastic strain values of 0.5 and 1.5 (these are also parameters that can be set). Therefore, when we defined a random values of plastic strain in the central region of the model, we also changed the friction and viscosity values compared to their reference value. This helps to localise deformation and forming faults-like structures. Higher values of weakening factors will reduce less the viscosity, whereas lower values will have a higher impact on decreasing the viscosity through the brittle deformation mechanism. 
+An other thing worth noting is that we use an harmonic average scheme for the way material properties are averaged over the grid. This allows for a smooth and stable solution, but does create less sharp contrasts at the interface between different materials. 
+(VM - Maybe no need to add the entire material model, only some parts?)
```

Comment:
I think only adding a few parts is fine for now, we can always revisit later on.